mtp_cashbook/assets-src/stylesheets/views/_batch.scss

```diff
@@ -3,12 +3,17 @@
   margin-bottom: 1em;
 
   p {
-    display: inline;
     font-size: 24px;
+    margin-top: 0;
+
+    @include breakpoint(769px) {
```

Comment:
769 is the dimension GDS use as desktop width. This is using @tyom's new mojular and I don't think it has a way of clearly showing it's a desktop width.

I can look at moving these values into something a bit clearer.